ipaclient/install/client.py

```diff
@@ -1418,12 +1418,18 @@ def do_nsupdate(update_txt):
 
     result = False
     try:
-        ipautil.run([paths.NSUPDATE, '-g', UPDATE_FILE])
+        if dns_over_tls:
+            ipautil.run([paths.NSUPDATE, '-p', '853', '-S', '-g', UPDATE_FILE])
```

Comment:
What's the content of the update file?

This will probably not work, must be tested. See: https://gitlab.isc.org/isc-projects/bind9/-/issues/5028

So far, you need to set -H and server command.

Reply:
Yes, it gets set in ipaclient/install/client.py::configure_sssd_conf

Reply:
Ok, that needs to be updated see https://github.com/SSSD/sssd/blob/d2232139a2b9453f28824714c8b04b9e9f554df9/src/man/sssd-ipa.5.xml#L314

Since nsupdate is currently buggy (and server autodiscovery is not done via DoT at all) you need to hardcode dns in dyndns_server using dot uri.

Reply:
Thanks for your comment, just updated the PR.